### DISCUSSION TITLE
使用dae 作为透明网关 访问国内地址很慢，查了ip(国内) 和 日志规则(显示直连)

### DISCUSSION THREAD

**Discussion by linocz1** [daeuniverse/dae#519] [General]
这是我的配置文件

# 全局配置
global {
    # 绑定lan网口提供给内网设备，填自己服务器的网口，多个逗号隔开
    lan_interface: ens34
    wan_interface: auto
    log_level: info
    auto_config_kernel_parameter: true
    dial_mode: domain
    allow_insecure: false
    so_mark_from_dae: 1234

    # tls配置
    tls_implementation: utls
    utls_imitate: chrome_auto
}

# 订阅配置
subscription {
}

# 节点配置
node {
    # 节点URI
    'socks5://192.168.5.3:18080'
}

# 分组配置，以下采用固定策略，使用第一个节点
group {
    elden_proxy {
        policy: fixed(0)
    }
}

# dns配置
dns {
  upstream {
    googledns: 'tcp://dns.google.com:53'
    alidns: 'udp://dns.alidns.com:53'
  }
  routing {
    request {
      qname(geosite:cn) -> alidns
      fallback: googledns
    }
    response {
        upstream(googledns) -> accept
        fallback: accept
    }
  }
}

# 路由配置
routing {
    pname(NetworkManager) -> direct
    dip(224.0.0.0/3, 'ff00::/8') -> direct
    dip(geoip:private) -> direct

    ### 以下是自定义规则
  
    ## 规则
    ip(geoip:cn) -> direct
    domain(geosite:cn) -> direct
    domain(geosite:category-ads) -> block
 
  
    # 默认出站分组
    fallback: elden_proxy
}

**Comment by linocz1** [comment]
已解决

**Reply by ShutongZHENG** [reply]
怎么解决的求！

**Comment by xu1919** [comment]
怎么解决的？说一下嘛
